app/imageswap/test/test_exact_mapping.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python
+
+# Copyright 2020 The WebRoot, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import unittest
+from unittest.mock import patch
+
+sys.path.append("./app/imageswap")
+import imageswap
+
+###########################################################################
+# Test image tag mapping scenarios ########################################
+###########################################################################
+
+
+@patch("imageswap.imageswap_mode", "MAPS")
+@patch("imageswap.imageswap_maps_file", "./testing/map_files/map_file.conf")
+class ExactMapping(unittest.TestCase):
+    def setUp(self):
+
+        self.app = imageswap.app.test_client()
+        self.app.testing = True
+        imageswap.app.logger.setLevel("DEBUG")
+
+    def tearDown(self):
+
+        pass
+
+    def test_map_default(self):
+
+        """Method to test that default mapping is working"""
```

Comment:
nitpick (blocking): Several of the method descriptions (ie. comment inside of `"""`) look like they may have been copied/pasted form other tests. Try and make sure the description matches the test case. These can end up being wordy, but they help when looking at the unites output as most of these are very similar with only slight mods per test. This is applicable to most, if not all descriptions in this file. 

Make sure all of these include something that mentions the `Exact` mapping/syntax.

One example from the library image mapping tests:

```
 """Method to test Map File config (map with library registry and a tag that contains a ".")"""
```

Please let me know if you have questions on this.

Reply:
I updated the comments, please take a look at it, whether it is now better explaining the testcase.